esp/applications/eclwatch/testauth_authorization_map.yaml

```diff
@@ -0,0 +1,43 @@
+authNZ:
+   test:
+      name: test
+      type: test
+      LibName: libtestauthSecurity.so
+      authtype: AuthTypeMixed
+      # Define which authorization resources to be tested.
+      # As default, the authorization resources defined in
+      # ldap_authorization_map.yaml are tested.
+      useResourceMapsFrom: ldap
+      useraccess:
+      -  username: testadmin
+         #secretkey: testauthusercreds1     # optional
+         #vaultkey: testauthuservaultcreds1 # optional
+         defaults:
+            filescope: FULL
```

Comment:
minor: lower case for full and none would be more common in a yaml file.

Reply:
Will fix